### DISCUSSION TITLE
uninstall in AIR package

### DISCUSSION THREAD

**Discussion by balaedit** [airsdk/Adobe-Runtime-Support#2541] [General]
While packing the package for windows as an AIR package, how to include uninstall file in it.

**Comment by ajwfrost** [comment]
If you mean a .air file, then when this is installed on a computer by the AIR shared runtime/application installer process, it will automatically generate the appropriate entries in the Add/Remove Programs system. So, you can uninstall from there.

You can also just run the .air file again, and it will tell you it's already installed and give you an option to uninstall it.

**Reply by balaedit** [reply]
Other softwares provide uninstall.exe likewise, can we provide uninstall.exe after installing the package.air file in the package or while packing air file some flag should be given to create uninstall.exe file after installing any air application.

**Reply by ajwfrost** [reply]
Some software apps come with an 'uninstall.exe' (particularly if they've been turned into MSI files using InstallSheild) but it's not actually necessary at all..

It should be possible to create a simple/standard .exe file that just calls the AIR application installer for this, but this is going to be low priority, I think most Windows users would be familiar with "add/remove programs"..

**Reply by balaedit** [reply]
Thanks, I know it's not much required but it required in my case. Keep it in low priority as the requirements is optional. Thanks @ajwfrost
